package/yast2.spec

```diff
@@ -19,114 +19,92 @@
+# for ag_anyxml
+Requires:       perl-XML-Simple
+# for defining abstract methods in libraries
+Requires:       rubygem(%{rb_default_ruby_abi}:abstract_method)
+# for file access using augeas
+Requires:       rubygem(%{rb_default_ruby_abi}:cfa)
+# for running scripts
+Requires:       rubygem(%{rb_default_ruby_abi}:cheetah)
 Requires:       sysconfig >= 0.80.0
+# ag_ini section_private
+# ag_ini with (un)quoting support
+Requires:       yast2-core >= 2.23.0
 Requires:       yast2-hardware-detection
-Requires:       yast2-xml
 # for SLPAPI.pm
 Requires:       yast2-perl-bindings
-# for ag_anyxml
-Requires:       perl-XML-Simple
-# for GPG.ycp
-Requires:       gpg2
-
-# for Punycode.rb (bnc#651893) - the idnconv tool is located in
-# different packages (SLE12/Leap-42.1: bind-utils, TW/Factory: idnkit)
-%if 0%{?suse_version} >= 1330
-Requires:       idnkit
-%else
-Requires:       bind-utils
-%endif
-
+# changed StartPackage callback signature
+Requires:       yast2-pkg-bindings >= 2.20.3
+Requires:       yast2-ruby-bindings >= 3.1.33
+Requires:       yast2-xml
+# new UI::SetApplicationIcon
+Requires:       yast2-ycp-ui-bindings >= 3.1.8
+Requires:       yui_backend
+# pre-requires for filling the sysconfig template (sysconfig.yast2)
+PreReq:         %fillup_prereq
 # xdg-su in .desktops
 Recommends:     xdg-utils
-
-# moved cfg_security.scr
-Conflicts:      yast2-security <= 2.13.2
+Conflicts:      autoyast2-installation < 2.16.2
+# country_long.ycp and country.ycp moved to yast2
+Conflicts:      yast2-country < 2.16.3
+# SrvStatusComponent moved to yast2.rpm
+Conflicts:      yast2-dns-server < 3.1.17
 # moved ag_netd, cfg_netd.scr, cfg_xinetd.scr
 Conflicts:      yast2-inetd <= 2.13.4
-Conflicts:      yast2-tune < 2.15.6
-Obsoletes:      yast2-mail-aliases <= 2.14.0
-Conflicts:      yast2-storage < 2.16.4
-Conflicts:      yast2-network < 2.16.6
-Conflicts:      yast2-sshd < 2.16.1
-
-# moved ag_content agent 
+# InstError
+Conflicts:      yast2-installation < 2.18.5
+# moved ag_content agent
 Conflicts:      yast2-instserver <= 2.16.3
-
 # moved cfg_mail.scr
 Conflicts:      yast2-mail < 3.1.7
-
-# InstError
-Conflicts:      yast2-installation < 2.18.5
-
-Conflicts:      yast2-update < 2.16.1
+Conflicts:      yast2-mouse < 2.16.0
+Conflicts:      yast2-network < 2.16.6
 # Older packager use removed API
 Conflicts:      yast2-packager < 3.1.34
-Conflicts:      yast2-mouse < 2.16.0
-Conflicts:      autoyast2-installation < 2.16.2
-# country_long.ycp and country.ycp moved to yast2
-Conflicts:      yast2-country < 2.16.3
-# SrvStatusComponent moved to yast2.rpm
-Conflicts:      yast2-dns-server < 3.1.17
-
+# moved cfg_security.scr
+Conflicts:      yast2-security <= 2.13.2
+Conflicts:      yast2-sshd < 2.16.1
+Conflicts:      yast2-storage < 2.16.4
+Conflicts:      yast2-tune < 2.15.6
+Conflicts:      yast2-update < 2.16.1
+Obsoletes:      yast2-mail-aliases <= 2.14.0
 Provides:       yast2-lib-sequencer
 Obsoletes:      yast2-lib-sequencer
```

Comment:
those provides and obsoletes can be also removed. It is ancient noone should use it. So whole Provides and Obsoletes part can be completely removed.